www-apps/redmine/redmine-3.2.7-r1.ebuild

```diff
@@ -0,0 +1,215 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Please say what you're fixing in the summary line. Bug numbers are not very useful for someone looking through `git log`. You can mention it in footer using `Bug: https://bugs.gentoo.org/NNNNNN`.